cluster-controller/src/test/java/io/strimzi/controller/cluster/ResourceUtils.java

```diff
@@ -58,8 +59,8 @@ private ResourceUtils() {
      * @param healthTimeout
      * @return
      */
-    public static ConfigMap createConfigMap(String clusterCmNamespace, String clusterCmName, int replicas, String image, int healthDelay,
-                                            int healthTimeout, String metricsCmJson) {
+    public static ConfigMap createKafkaClusterConfigMap(String clusterCmNamespace, String clusterCmName, int replicas, String image, int healthDelay,
```

Comment:
Long line

(I'm sure I'm guilty of plenty of long lines too, but since it's helpful when reviewing if you don't have to scroll I'm happy to be picked up on it)